lib/ionic_notification/push_service.rb

```diff
@@ -3,35 +3,19 @@ class PushService
     include HTTParty
     base_uri IonicNotification.ionic_api_url
 
-    attr_accessor :device_tokens, :message
+    attr_accessor :body
 
-    def initialize(**args)
-      #args.each &method(:instance_variable_set)
-      args.each do |attr, value|
-        instance_variable_set("@#{attr}", value)
-      end
+    def initialize(body)
+      @body = body
     end
 
     def notify!
       self.class.post("/api/v1/push", payload)
     end
 
-    def alert!(msg)
-      notify do
-        {
-          alert: msg
-        }
-      end
-    end
-
-    def notify(&block)
-      @message =  yield(block)
-      notify!
-    end
-
     def payload
       options = {}
-      options.merge!(body: body).merge!({ basic_auth: auth}).merge!({ headers: headers})
+      options.merge!(body: @body).merge!({ basic_auth: auth}).merge!({ headers: headers})
```

Comment:
Line is too long. [89/80]<br>Redundant curly braces around a hash parameter.<br>Space inside } missing.